Dockerfile.spdk

```diff
@@ -49,6 +50,16 @@ RUN \
 # If we run from github disable also VPCLMULQDQ as some of github's runners don't have it
 RUN \
     if [[ -n "$SPDK_DISABLE_VPCLMULQDQ" ]]; then sed -i "s/^\( \+'VPCLMULQDQ'\)/#\1/" dpdk/config/x86/meson.build ; fi
+RUN \
+    if [[ -n "$SPDK_DISABLE_AVX512" ]]; then sed -i "s/^\( \+'AVX512BW'\)/#\1/" dpdk/config/x86/meson.build ; fi
+RUN \
+    if [[ -n "$SPDK_DISABLE_AVX512" ]]; then sed -i "s/^\( \+'AVX512CD'\)/#\1/" dpdk/config/x86/meson.build ; fi
+RUN \
+    if [[ -n "$SPDK_DISABLE_AVX512" ]]; then sed -i "s/^\( \+'AVX512DQ'\)/#\1/" dpdk/config/x86/meson.build ; fi
+RUN \
+    if [[ -n "$SPDK_DISABLE_AVX512" ]]; then sed -i "s/^\( \+'AVX512F'\)/#\1/" dpdk/config/x86/meson.build ; fi
+RUN \
+    if [[ -n "$SPDK_DISABLE_AVX512" ]]; then sed -i "s/^\( \+'AVX512VL'\)/#\1/" dpdk/config/x86/meson.build ; fi
```

Comment:
nit:  how about employing a single sed run with a regular expression that covers all instances of `AVX512*`? For instance, something like: re: `AVX512(BW|CD|DQ|F|VL)`.

Reply:
@baum thought about that, didn't want to use too complicated expressions. I'll have another look.

Reply:
Changed it to:

```
RUN \
    if [[ -n "$SPDK_DISABLE_AVX512" ]]; then sed -i "s/^\( \+'AVX512\)/#\1/" dpdk/config/x86/meson.build ; fi
```